hnn_core/network.py

```diff
@@ -518,3 +518,32 @@ def plot_spikes(self, ax=None, show=True):
         if show:
             plt.show()
         return ax.get_figure()
+
+    def write_spikes(self, fname, trial_idx='all'):
+        """Write spike times to a file.
+
+        Parameters
+        ----------
+        fname : str
+            Full path to the output file (.txt)
+        trial_idx : list of int
+            Indices of selected trials. If 'all',
+            all trials are selected.
+        """
+        if trial_idx=='all':
+            trial_idx = range(len(self.spiketimes))
+
+        spiketimes = []
+        spikegids = []
+        for i in trial_idx:
```

Comment:
avoid use of single letter variable names when possible. They are hard to Ctrl + F